easybuild/easyconfigs/g/gnuplot/gnuplot-6.0.1-GCCcore-13.2.0.eb

```diff
@@ -41,6 +41,9 @@ preconfigopts += 'export LDFLAGS="-Wl,--copy-dt-needed-entries" && '
 
 configopts = '--with-qt=qt6 --without-latex --disable-wxwidgets'
 
+buildopts = 'LDFLAGS="$LDFLAGS -Wl,--copy-dt-needed-entries" '
+buildopts += 'LIBS="-lcerf -lcairo -lpangocairo-1.0"'
+
```

Comment:
Are these changes necessary to get `Octave` to work or is this just a general improvement for `gnuplot`?